server/backend/quiz/views.py

```
@@ -56,6 +56,16 @@ def post(self, request):
             return Response(serializer.data, status=status.HTTP_201_CREATED)
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+    
+from typing import ClassVar
+
+class UpdateDeleteQuizView(APIView):
+    permission_classes: ClassVar = [IsAuthenticated, IsAdminUser]
+
+    def delete(self, request, category_id: int) -> Response:
```

Ruff (0.8.2)

Missing type annotation for function argument `request`

(ANN001)